### DISCUSSION TITLE
Adding a spoiler to a blog post?!

### DISCUSSION THREAD

**Discussion by xDGameStudios** [mmistakes/minimal-mistakes#2927] [Q&A] [answered]
Is there a way to add a spoiler to a blog post inside markdown?
I'm using:

```html
<details>
  <summary>Click to Check Your Answer!</summary>
    // want to show a codeblock here!
</details>
```
it works creating the spoiler but then I not able to place a code inside using markdown,

**Comment by mmistakes** [comment, marked as answer]
Limitation of Markdown. You can't place Markdown inside of HTML.
You can do it with some Liquid tags (e.g. [`capture`](https://shopify.github.io/liquid/tags/variable/) and [`markdownify`](https://jekyllrb.com/docs/liquid/filters/) filter) though, but it's not all that clean.

```
{% capture details %}
Place some **markdown** here if you like.

Can be multiple paragraphs, `inline code`, whatever.
{% endcapture %}

<details>
  <summary>Click to check your answer!</summary>
  {{ details | markdownify }}
</details>
```

**Reply by xDGameStudios** [reply]
That's good enough for me 😄
